click/base.js

```diff
@@ -1,5 +1,7 @@
 console.log("Sanity Check: JS is working!");
 
-$(document).ready(function(){
+$('#imperatives span').on("click", function handleClick(event){
 
-})
+      $('#imperatives ul').append("<li>''#imperatives'</li>");
+      console.log(this);
```

Comment:
This will append the string '#imperatives' to your page.  You want to append the text contained in the clicked span.